### ISSUE TITLE
Last Played date is null

### ISSUE THREAD

**Issue by sterling-tenn** [rombat/musicbee-navidrome-sync#9]
Hello, I've encountered an issue and I hope to hear your advice.

I've been able to successfully run a `fullSync` to my `navidrome.db` (see [this issue](https://github.com/rombat/musicbee-navidrome-sync/issues/5#issuecomment-1992684142) for extra details) and everything appears to be working properly with the exception of the "Last Played" date as seen here:
![image](https://github.com/rombat/musicbee-navidrome-sync/assets/56081843/d745f4a5-46ac-49d0-bbf8-44e78563157b)

Digging a bit further into the source code, logging the value of `musicBeeCollection` in `MBNDSynchronizer.js` after it's assignment on [line 191](https://github.com/rombat/musicbee-navidrome-sync/blob/1f7114dac9f06fd8779dd3ec7eed0306574a41f8/lib/handlers/MBNDSynchronizer.js#L191), the output of a single entry looks something like this
```
{
  filePath: 'C:\\Users\\sterl\\OneDrive\\Music',
  filename: '[.que] - rest.mp3',
  folder: 'Music',
  title: 'rest',
  lastPlayed: null,
  love: 0,
  skipCount: 0
}
```
Where the `lastPlayed` value is `null`. My first guess is something to do with grabbing the date value from the `MusicBee_Export.csv` on [line 167](https://github.com/rombat/musicbee-navidrome-sync/blob/1f7114dac9f06fd8779dd3ec7eed0306574a41f8/lib/handlers/MBNDSynchronizer.js#L167C9-L167C120)?

For reference a truncated version of my `MusicBee_Export.csv` looks like this:
```
"<File Path>","<Filename>","<Folder>","Title","Last Played","Play Count","Rating","Love","Skip Count"
"C:\Users\sterl\OneDrive\Music","[.que] - rest.mp3","Music","rest","2023-10-24 23:50","4","","","0"
```

Thank you

**Comment by rombat** [comment]
Hi
I'm sorry to answer this late, I'm really busy at work these past weeks, and I totally missed your issue.
I'll have a look as soon as I can and answer you properly afterwards, thank you for your patience 🙏🏼

Is the lastPlayed property missing from all of your tracks, or just some ? If so, can you provide a more complete csv, including tracks that don't have this property?
Thanks in advance

**Comment by rombat** [comment]
I think I can see why it's null, I think it's a matter of date format. I'm checking all this on my phone, so I'll catch you up later when I can access my computer.

**Comment by rombat** [comment]
@sterling-tenn I've fixed this by adding a new option (`--datetime-format <format>`) to pass a specific datetime format. `DD/MM/YYYY HH:mm` was used, but yours is `YYYY-MM-DD HH:mm`, I wasn't aware that MusicBee could output different formats for datetime.
I've released [a new version](https://github.com/rombat/musicbee-navidrome-sync/releases/tag/v1.1.0) with this option and updated the readme accordingly.

Regarding your other issue with path, I'll fix that properly later, but thanks for the input 👍🏼
